crates/rpc/rpc-types/src/eth/call.rs

```diff
@@ -30,8 +30,8 @@ pub struct EthCallResponse {
     #[serde(skip_serializing_if = "Option::is_none")]
     /// eth_call output (if no error)
     pub output: Option<Bytes>,
-    /// eth_call error output
-    pub error: Option<Bytes>,
+    /// eth_call output (if error)
+    pub error: Option<String>,
```

Comment:
are these fields mutually exclusive?
like if `output`, then error is missing?

if so this field also needs the
#[serde(skip_serializing_if = "Option::is_none")]

annotation

Reply:
Ah okay I thought that adding that to the top of the struct would apply it to all fields, but that makes sense!